### ISSUE TITLE
can you provide the pretrained model?

### ISSUE THREAD

**Issue by ccqedq** [hrlblab/CircleNet#30]

**Comment by hrlblab** [comment]
Yes, the journal paper is under-review. We will release that once the paper is accepted

**Comment by ccqedq** [comment]
Isn't CircleNet already accepted by MICCAI 2020 and IEEE TMI 2021?

**Comment by ccqedq** [comment]
Can this pretrained model generalize to detect common circular targets? I am concerned that the results trained on a specific dataset may not perform well on other datasets, and as you know, medical image data is difficult to collect

**Comment by hrlblab** [comment]
I don't think so. That will be data specific
